recurly/resource.py

```diff
@@ -714,8 +714,8 @@ def to_element(self, root_name=None):
                 value = self.__dict__[attrname]
             except KeyError:
                 continue
-
-            if attrname in self.xml_attribute_attributes:
+            # For billing_info, type property takes precedence over xml attribute. 
+            if attrname in self.xml_attribute_attributes and attrname != 'type' and root_name != 'billing_info':
```

Comment:
If you could, double check that this is still working with other scripts (creating / updating credit cards, etc)